guests/templates/checklist/band_merch_deadline.html

```diff
@@ -35,36 +132,235 @@
-      merchandise from our own dedicated table in the future.</label>
-      <br/> <br/>
+      <div class="form-group">
+        <div class="col-sm-9 col-sm-offset-3 checkbox">
+          <label class="checkbox-label">
+            <input type="checkbox" name="coverage" value="yes" />
+            I commit to staffing our table in the marketplace at least <b>8 hours per day</b>.
+          </label>
+        </div>
+      </div>
+      <div class="form-group">
+        <div class="col-sm-9 col-sm-offset-3 checkbox">
+          <label class="checkbox-label">
+            <input type="checkbox" name="warning" value="yes" />
+            I acknowledge that failing to staff our table for <b>8 hours per day</b> will revoke
+            the privilege of selling our merchandise from our own dedicated table in the future.
+          </label>
+        </div>
+      </div>
     </div>
+  {% endif %}
 
-    <script>
-        var checkCoverage = function () {
-            var dedicated = $.val('selling_merch') === {{ c.OWN_TABLE }};
-            setVisible('#coverage', dedicated);
-
-            var allChecked = $.field('coverage').prop('checked') && $.field('warning').prop('checked');
-            $('#submit').prop('disabled', dedicated && !allChecked);
-        };
-        $(function () {
-            checkCoverage();
-            $.field('selling_merch').on('change', checkCoverage);
-            $.field('warning').on('click', checkCoverage);
-            $.field('coverage').on('click', checkCoverage);
-        });
-    </script>
+  {% if c.ROCK_ISLAND %}
+    <div id="rockisland" style="display:none">
+      <h3>Your Inventory</h3>
+      <div id="inventory_container"{% if not guest_merch.inventory %} style="display:none"{% endif %}>
+        {{ guests_macros.inventory_table(show_controls=True) }}
+      </div>
+      <div class="well well-sm well-success col-sm-6 col-sm-offset-3 text-success text-center">
+        <h4>Add some merch to your inventory!</h4>
+        <div class="inventory-add-item text-success">
+          <span class="glyphicon glyphicon-plus-sign"></span>
+          <select class="form-control add_inventory">
+            <option value="">Select an item to add it...</option>
+            {{ options(c.MERCH_TYPES_OPTS) }}
+          </select>
+        </div>
+      </div>
+      <div class="clearfix"></div>
+      <h3>Merch Point of Contact</h3>
+      {{ guests_macros.name_form_group(guest_merch, first_name_attr='poc_first_name', last_name_attr='poc_last_name', is_required=True) }}
+      {{ macros.form_group(guest_merch, 'poc_phone', label='Phone Number', type='tel', placeholder='555-555-1234', is_required=True) }}
+      {{ macros.form_group(guest_merch, 'poc_email', label='Email', type='email', placeholder='merch@example.com', is_required=True) }}
+      {{ macros.form_group(guest_merch, 'poc_mailing_address', label='Mailing Address', placeholder='123 Main St, Anytown 12345', is_required=True) }}
```

Comment:
We have a really great mailing form macro, with the ability to have a prefix and everything. Any particular reason you skipped using it?

Reply:
Yep! Laziness on my part!